.travis.yml

```diff
@@ -11,6 +11,7 @@ gemfile:
   - gemfiles/activerecord_3.2.gemfile
   - gemfiles/activerecord_4.0.gemfile
   - gemfiles/activerecord_4.1.gemfile
+  - gemfiles/activerecord_4.2.gemfile
```

Comment:
yeah, trying to keep the diff low until it's green :D

On Tue, Aug 18, 2015 at 2:11 PM, Pierre Schambacher <
notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/zendesk/kasket/pull/16#discussion_r37354129>:
>
> > @@ -11,6 +11,7 @@ gemfile:
> >    - gemfiles/activerecord_3.2.gemfile
> >    - gemfiles/activerecord_4.0.gemfile
> >    - gemfiles/activerecord_4.1.gemfile
> > +  - gemfiles/activerecord_4.2.gemfile
>
> Ruby 1.9.3 can probably die as well so we can use prepend
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/kasket/pull/16/files#r37354129>.
>